### PR TITLE
feat: request products from labda BE

### PR DESCRIPTION
**Pull request by chermic**
frontent app link - https://d3l9zqe2490dzq.cloudfront.net/

### CODE REVIEW

File: src/constants/apiPaths.ts [hunk 1]
```diff
@@ -1,9 +1,8 @@
-
 const API_PATHS = {
   product: 'https://.execute-api.eu-west-1.amazonaws.com/dev',
```
Comment (ilya-moskovtsev) [inline src/constants/apiPaths.ts:2]: If update other API paths as well, this page https://d3l9zqe2490dzq.cloudfront.net/admin/products and some others should show products.

**Review by ilya-moskovtsev — COMMENTED** [review]
https://github.com/Terr1bus/nodejs-aws-fe/pull/2/files#diff-86ffb3354e7dc44b80c2d618c41db4d73a22237306e7db3a17b624d431f23ef4R50
Product images:
- should not be randomly generated
- should be received from the backend

**Comment by ilya-moskovtsev** [comment]
https://github.com/Terr1bus/nodejs-aws-fe/pull/2/files#diff-86ffb3354e7dc44b80c2d618c41db4d73a22237306e7db3a17b624d431f23ef4R51
Image titles should be received from the backend

**Review by ilya-moskovtsev — APPROVED** [review]

File: src/constants/apiPaths.ts (hunk 1 above)
Comment (chermic) [reply src/constants/apiPaths.ts:2]: I think that this endpoints will be used in the future tasks, so i didn't touch them

**Comment by chermic** [comment]
> https://github.com/Terr1bus/nodejs-aws-fe/pull/2/files#diff-86ffb3354e7dc44b80c2d618c41db4d73a22237306e7db3a17b624d431f23ef4R51
> Image titles should be received from the backend

I'm not sure, in tasks not talked about this. I think that best way for delivery images is upload them at S3 bucket and share, but this is only my think :)

**Comment by chermic** [comment]
> https://github.com/Terr1bus/nodejs-aws-fe/pull/2/files#diff-86ffb3354e7dc44b80c2d618c41db4d73a22237306e7db3a17b624d431f23ef4R50
> Product images:
> 
> * should not be randomly generated
> * should be received from the backend

I think this is not nessesary at this task, but thank you for reminder :)

**Pull request merged**